.docker/config/solr/schema_extra_types.xml

```diff
@@ -419,10 +430,10 @@
   English Text Field
   7.0.0
 -->
-<fieldType name="text_en" class="solr.TextField" positionIncrementGap="100">
+<fieldType name="text_en" class="solr.TextField" positionIncrementGap="100" storeOffsetsWithPositions="true">
```

Comment:
@steveworley this change will disrupt all sites when they write into the existing index unless the deployment recreates the Solr core and reindexes the content. We will need to find a solution for sites with a high content volume.